roadmap.md

```diff
@@ -20,8 +20,9 @@ The PELUX project aims to provide biannual releases. The first release was done
 in January 2018. For already released versions, see the [downloads](/downloads)
 page.
 
-### PELUX 2.0 [July 2018]
+### PELUX 3.0 [September 2018]
 Content of this release is under discussion. Tentatively the topics in focus will be:
-- Intel ARP board support
-- Qt 5.10
-- SOTA updates
+- Adaptation to Yocto "sumo" 2.5
+- Neptune UI 3
+- Qt 5.11
+- Obviously various fixes
```

Comment:
I would prefer to remove the "Obviously", yes it is obvious, but "Various fixes" is enough in my opinion.